src/state/axisSelectors.ts

```diff
@@ -1,55 +1,136 @@
 import { createSelector } from '@reduxjs/toolkit';
-import { selectChartLayout } from '../context/chartLayoutContext';
-import { ParsedScaleReturn, parseScale } from '../util/ChartUtils';
-import { AxisDomain, AxisType, CategoricalDomain, LayoutType, NumberDomain } from '../util/types';
-import { AxisId, AxisSettings } from './axisMapSlice';
-import { selectChartName } from './selectors';
+import range from 'lodash/range';
```

Comment:
people don't like that we use lodash, but I guess idc since maintaining our own implementation would suck

Reply:
I copied from the previous place, the lodash question is not really the topic of the day now. We can later replace it with d3 or our own impl.